code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1160,6 +1150,63 @@ var/list/rank_prefix = list(\
 	else
 		return 0
 
+#define MODIFICATION_ORGANIC 1
+#define MODIFICATION_SILICON 2
+#define MODIFICATION_REMOVED 3
+
+//Needed for augmentation
+/mob/living/carbon/human/proc/rebuild_organs(var/from_preference = 0)
+	if(!species)
+		return 0
+
+	for(var/obj/item/organ/organ in (organs|internal_organs))
+		qdel(organ)
+
+	if(organs.len)                  organs.Cut()
```

Comment:
Condition expressions should not be inline, please use multiline style (pr-wide comment).